cli/cmd/encore/eject.go

```diff
@@ -49,18 +50,26 @@ func init() {
 	dockerEjectCmd.Flags().StringVar(&p.Goos, "os", p.Goos, "target operating system. One of (linux, darwin, windows)")
 	dockerEjectCmd.Flags().StringVar(&p.Goarch, "arch", p.Goarch, "target architecture. One of (amd64, arm64)")
 	dockerEjectCmd.Flags().BoolVar(&p.CgoEnabled, "cgo", false, "enable cgo")
+	dockerEjectCmd.Flags().BoolVar(&p.SkipInfraConf, "skip-infra-config", false, "do not read or generate a infra configuration file")
```

Comment:
If the argument is `--config`, should this be `--skip-config` instead?

Reply:
yes!